shared/fs/common/path-item-action/menu.js

```diff
@@ -55,6 +55,7 @@ const hideMenuOnClick = (onClick: (evt?: SyntheticEvent<>) => void, hideMenu: ()
 }
 
 const makeMenuItems = (props: Props, hideMenu: () => void) => [
+  'Divider',
```

Comment:
Somehow this didn't show up for me on mobile. I guess it's just not supported on mobile. Perhaps this divider should be part of the header?

Reply:
It looks like we intentionally strip out dividers on mobile for some reason: https://github.com/keybase/client/blob/b4cb5dc540dac5126a783a8afeaf0374073c3d07/shared/common-adapters/floating-menu/menu-layout/index.native.js#L53-L58

Reply:
Regardless, it feels like there is a gap on mobile (er, on Android anyway) by virtue of the way the menus work over there. @cecileboucheron was the gap in KBFS-4023 supposed to be added on both mobile and desktop, or just desktop?

Reply:
Weird... There should be a divider on both desktop and mobile between the info and the actions.

Reply:
Okay, I'm changing the code to allow dividers as the first element on mobile in a FloatingMenu.